charts/feast/templates/core-deploy.yaml

```diff
@@ -57,6 +57,12 @@ spec:
           limits:
             cpu: {{ .Values.core.resources.limits.cpu }}
             memory: {{ .Values.core.resources.limits.memory }}
+        {{- if .Values.serviceAccount }}
+        volumeMounts:
+          - mountPath: "/etc/gcloud/service-accounts"
+            name: "{{ .Values.serviceAccount.name }}"
+            readOnly: false
+        {{- end }}
         env:
```

Comment:
The following new env vars are required:
- `DATASET_PREFIX`: dataset name prefix, defaulting to `fs`
-`JOB_ERRORS_STORE_TYPE`: job errors store type, defaulting to `stderr` or `stdout`
- `JOB_ERRORS_STORE_OPTIONS`: options for job errors store, defaulting to an empty json string `{}`